src/qiskit_iqm/fake_backends/fake_adonis.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022-2023 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fake (i.e. simulated) backend for IQM Adonis QPU."""
+from iqm_client import QuantumArchitectureSpecification
+
+from qiskit_iqm.fake_backends.chip_sample import IQMChipSample
+from qiskit_iqm.fake_backends.iqm_fake_backend import IQMFakeBackend
+
+
+class IQMFakeAdonis(IQMFakeBackend):
+    """
+    Fake backend for simulating an IQM Adonis QPU.
```

Comment:
This is an Adonis with default noise params. If you want an Adonis with custom noise params, you'll have to use IQMFakeBackend directly, since `__init__` does not take an IQMChipSample argument. Is this intentional?

Reply:
I think it is somewhat intentional. If one wants to create fake backend with specific parameters they should use IQMFakeBackend. This class is just a helper to get a quick access to some default fake backend instance without caring about anything